src/chromium/interfaces/contextMenus.js

```diff
@@ -112,51 +114,38 @@ export async function handlePlatformContextMenuClick(info, tab) {
 
   if (info.menuItemId === "changeDefaultLaunchContextMenu") {
     await handleChangeDefaultLaunchContextMenuClick();
-    browser.storage.local.set({
-      telemetry: {
-        type: "currentBrowserChange",
-        from: externalBrowserName,
-        to:
-          externalBrowserName === "Firefox"
-            ? "Firefox Private Browsing"
-            : "Firefox",
-        source: "action_context_menu",
-      },
-    });
+    settingEvent.currentBrowser.from.set(externalBrowserName);
```

Comment:
I haven't looked at the library, so I am wondering - is this the right/documented way to set multiple properties of one event at once?

Reply:
Initglean is required once on startup and does nothing on subsequent calls after being initialized. I only have it within the install and startup listeners to ensure I don't miss a case.

To set properties, I'm using the[ labeled string metric](https://mozilla.github.io/glean/book/reference/metrics/labeled_strings.html). I don't immediately see a way to set multiple at once.

Reply:
Although now looking through it, perhaps using the [event metric](https://mozilla.github.io/glean/book/reference/metrics/event.html) instead of a labeled string makes more sense.